arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/Sequence.kt

```diff
@@ -108,11 +109,13 @@ fun <E, A> Sequence<Either<E, Sequence<A>>>.flatSequenceEither(): Either<E, Sequ
 fun <E, A> Sequence<Validated<E, Sequence<A>>>.flatSequenceValidated(semigroup: Semigroup<E>): Validated<E, Sequence<A>> =
   flatTraverseValidated(semigroup, ::identity)
 
+@Deprecated(TraverseDeprecation)
```

Comment:
Idem for these, since this file doesn't exist in 0.11.0 so we can simply remove them all.